syne_tune/optimizer/schedulers/hyperband.py

```diff
@@ -73,11 +73,7 @@
     "register_pending_myopic": False,
     "do_snapshots": False,
     "rung_system_per_bracket": False,
-    "rung_system_kwargs": {
```

Comment:
This is removing functionality. Fine with me, but are you sure?

Reply:
I'd remove it if it is not complete, but if the features are fully implemented there is no issue leaving them in.

Reply:
Yes, we need this also for another forthcoming varianrt